### PR TITLE
#212: Make endpoint selectable for graphiql

### PR DESCRIPTION
**Pull request by Schedulaar**

### CODE REVIEW

File: backend/src/main/resources/graphiql/renderExample.js [hunk 1]
```diff
@@ -11,139 +11,150 @@
...
-      onEditQuery: onEditQuery,
-      onEditVariables: onEditVariables,
-      onEditHeaders: onEditHeaders,
-      defaultSecondaryEditorOpen: true,
-      onEditOperationName: onEditOperationName,
-      headerEditorEnabled: true,
-  }),
-  document.getElementById('graphiql'),
-);
+ReactDOM.render(React.createElement(SelectGraphqlInstance), document.getElementById('graphiql'));
```
Comment (elKei24) [inline backend/src/main/resources/graphiql/renderExample.js:160]: Why is there another call to `React.createElement` necessary? Does not `SelectGrapphqlInstance` already create an element?

**Review by elKei24 — APPROVED** [review]

File: backend/src/main/resources/graphiql/renderExample.js (hunk 1 above)
Comment (Schedulaar) [reply backend/src/main/resources/graphiql/renderExample.js:160]: `SelectGraphqlInstance` itself is only a function which cannot be passed to `ReactDOM.render`.

**Comment by Schedulaar** [comment]
On Hold until discussion about GraphQL tactic is resolved...

File: backend/src/main/resources/graphiql/renderExample.js (hunk 1 above)
Comment (elKei24) [reply backend/src/main/resources/graphiql/renderExample.js:160]: Ah, of course, thanks for the explanation 👍🏽

**Comment by Schedulaar** [comment]
Closed, as we decided for going the single graphql endpoint way for now...

**Pull request closed without merging**